panther_lights_controller/src/controller_node.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+
+import os
+import yaml
+from queue import Queue
+from threading import Lock
+
+import rospkg
+import rospy
+
+from panther_msgs.srv import SetLEDAnimation, SetLEDAnimationRequest, SetLEDAnimationResponse
+from panther_msgs.srv import SetLEDBrightness, SetLEDBrightnessRequest, SetLEDBrightnessResponse
+from panther_msgs.srv import (
+    SetLEDImageAnimation,
+    SetLEDImageAnimationRequest,
+    SetLEDImageAnimationResponse,
+)
+
+import panther_apa102_driver
+from animation.image_animation import ImageAnimation
+
+
+MAX_BRIGHTNESS = 31
+
+
+class PantherAnimation:
+    front: ImageAnimation
+    rear: ImageAnimation
+    interrupting = False
+
+
+class LightsControllerNode:
+    def __init__(self, name) -> None:
+        rospy.init_node(name, anonymous=False)
+
+        self._current_animation = None
+        self._default_animation = None
+        self._interrupt = False
+        self._animation_finished = True
+        self._anim_queue = Queue()
+        self._lock = Lock()
+        rospack = rospkg.RosPack()
+
+        default_animation_description = os.path.join(
+            rospack.get_path('panther_lights_controller') + '/config/panther_lights_animations.yaml'
+        )
+
+        self._animation_description = rospy.get_param(
+            'animation_description', default_animation_description
+        )
+
+        global_brightness = rospy.get_param('global_brightness', 1.0)
+        if 0.0 <= global_brightness <= 1.0:
+            apa_driver_brightness = int(global_brightness * MAX_BRIGHTNESS)
+        else:
+            rospy.logwarn(f'{rospy.get_name()} Brightness out of range <0,1>. Using default.')
+            apa_driver_brightness = MAX_BRIGHTNESS
+
+        self._num_led = rospy.get_param('num_led', 46)
+        self._controller_frequency = rospy.get_param('controller_frequency', 100)
+
+        # define controller and clear all panels
+        self._controller = panther_apa102_driver.PantherAPA102Driver(
+            num_led=self._num_led, brightness=apa_driver_brightness
+        )
+        self._controller.clear_panel(0)
+        self._controller.clear_panel(1)
```

Comment:
```suggestion
        self._controller.clear_panel(FRONT_PANEL)
        self._controller.clear_panel(REAR_PANEL)
```
It seems to me that linking the panel number to some verbal name will be more evocative.